hw/drivers/sensors/icp101xx/License.txt

```diff
@@ -0,0 +1,44 @@
+INVENSENSE, INC. (�LICENSOR�) SOFTWARE LICENSE AGREEMENT (�Agreement�)
+
```

Comment:
Thank you for the PR! 

This license is not an approved license and cannot be included in the repo, for a list of approved licenses please check out: https://apache.org/legal/resolved.html

Reply:
Licence updated for a BSD 3-Clause license